### ISSUE TITLE
train from scratch

### ISSUE THREAD

**Issue by jessicachen01** [nyukat/GMIC#30]
Hi,

Just wondering if the code for training can be released? and could you please provide some hint about how to train this model from scratch?

Thanks

**Comment by ChrisBtt** [comment]
I would also be interested in the training of this very promising network! 
Is there a possibility to get access to the code of the training loop with gradient descent?
